test/plugin/test_out_file.rb

```diff
@@ -183,6 +183,25 @@ def create_driver(conf = CONFIG, opts = {})
         Fluent::Test::Driver::Output.new(Fluent::Plugin::NullOutput).configure(conf)
       end
     end
+
+    test 'symlink path has not tag placeholder or key placeholder' do
+      conf = config_element('match', '**', {
+          'path' => "#{TMP_DIR}/${tag}/${key1}/${key2}/conf_test.%Y%m%d.%H%M.log",
+          'symlink_path' => "#{TMP_DIR}/conf_test.current.log",
+        }, [
+          config_element('buffer', 'time,tag,key1,key2', {
+              '@type' => 'file',
+              'timekey' => '1d',
+              'path' => "#{TMP_DIR}/buf_conf_test",
+          }),
+      ])
+      assert_nothing_raised do
+        d = create_driver(conf)
+        assert do
+          d.logs.count { |log| log.include?("symlink_path:") } == 2
```

Comment:
```suggestion
          d.logs.count { |log| log.include?("multiple chunks are competing for a single symlink_path") } == 2
```

It would be safer to look for a string that essentially represents the expected log.